src/petals/utils/peft.py

```diff
@@ -1,9 +1,13 @@
 import re
 import time
-from typing import List, Optional
+from typing import List, Optional, Sequence
```

Comment:
```suggestion
import os
from typing import List, Optional, Sequence

os.environ["BITSANDBYTES_NOWELCOME"] = "1"
```

Reply:
Since we do it here, please remove this line at `petals/utils/convert_block.py:59`.